tests/illumidesk/setup_course/test_course.py

```diff
@@ -0,0 +1,118 @@
+import os, sys
+import shutil
+from pathlib import Path
+import pytest
+from unittest.mock import patch
+from unittest.mock import Mock, MagicMock
+from docker.errors import NotFound
+from illumidesk.setup_course.course import Course
+
+
+@pytest.fixture(scope="function")
+def setup_environ(monkeypatch, tmp_path):
```

Comment:
Tests have pretty descriptive names, so kudos there. However lets add docstrings to add more specifics as to what the test is trying to accomplish, phrased as a question, for example: `Does the container start with an invalid value for the XYZ env var?`